FLAnimatedImage/BlendedImageCache.h

```diff
@@ -0,0 +1,20 @@
+//
+//  BlendedImageCache.h
+//  FLAnimatedImage
+//
+//  Created by Sih Ou-Yang on 2024/2/23.
+//  Copyright © 2024 com.cardinalblue. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@interface BlendedImageCache : NSObject
+
+- (instancetype _Nonnull)initWithLimit:(NSInteger)limit; // limit represents the size of the cache
+
+- (void)add:(UIImage *_Nonnull)image withIndex:(NSInteger)index;
```

Comment:
Have you considered adding the interface to remove the image?

```swift
func set(image: UImage?, at index: Int) 
func removeAll

// ..etc
```

Reply:
The advantage of removing all would be to allow the occupied memory to be purged more frequently.
I thought of removing all when the loop reaches the end. However, I hesitated to do it because the images will still be added back to the cache soon when the loop starts from the beginning again.

Reply:
Updated!